proposals/lip-0060.md

```diff
@@ -116,7 +116,7 @@ The block header is processed as follows:
   * Check that the block header follows the block header schema.
   * The value `b.header.version` can be any `uint32` integer.
   * The value of `b.header.transactionRoot` is equal to `EMPTY_HASH`.
-  * The value `b.header.assetsRoot` is validated as specified in [Update block schema and block processing LIP][research:update-block-schema#assets-root].
+  * The value `b.header.assetsRoot` is validated as specified in [LIP 0055][lip-0055#assets-root].
   * The value `b.header.timestamp` is Unix time in seconds and can be any value in the `uint32` range.
```

Comment:
Do we need to check that the timestamp "is Unix time in seconds"? Or do we imply only the check "can be any value in the `uint32` range"? 
Because the second check is always true and thus I would remove this line.

Reply:
I think it's nice to have the extra clarity

Reply:
I would also keep it as is.